### DISCUSSION TITLE
Is there any way to disable the AFK duration for the no longer AFK chat message?

### DISCUSSION THREAD

**Discussion by toydotgame** [sakura-ryoko/afkplus#22] [Q&A] [answered]
Hi, I'm moving over from the original afkdisplay mod as I've recently updated to 1.20.4. Thanks for maintaining it and updating it into its current form!

I liked the setup I had in afkdisplay where it was pretty minimal in comparison to the number of features. I'm able to get pretty close to afkdisplay's feel in-game by modifying the player name prefixes and setting the AFK reason to null, which removes the reason text from the "<Player> is now AFK" message:
```toml
defaultReason = ""
```
![image](https://github.com/sakura-ryoko/afkplus/assets/64190512/c1bf4618-eba6-4e00-b5c8-5268bd8f3e1d)

But I was wondering if there's a way to also remove the "Gone for:" text. It's a good feature and I like it but I'm aiming for the way I used to have it. Thanks!

**Comment by sakura-ryoko** [comment]
Ah good question, I only made it configurable in a sense of how the timer is displayed via the "prettyDuration" rather than allowing people to disable the feature...

It wouldn't be too complicated to just add another config option to disable the timer part of the message, and yes the "defaultReason" was an early addition when I created the "reason" handling, and that was intentional to allow the "" setting.

Perhaps it's something I can add for 1.20.5 for sure, unless you feel you want that option sooner.

I definitely was going for a configurable route that people can outright disable most features, at the complication of adding a bunch of settings for it.

**Comment by sakura-ryoko** [comment, marked as answer]
Feature is coming under the 1.4.7 release.

**Reply by toydotgame** [reply]
Thanks for adding this!

**Comment by toydotgame** [comment]
It seems that setting the `defaultReason` to null only removes the reason when the `/afk` command is run manually. When a player times out and an AFK message is sent, the reason "timeout" is appended to the message.

**Reply by sakura-ryoko** [reply]
See the 1.4.8 release, as this change allows for the "afkTimeoutString" to be set to "" 
https://github.com/sakura-ryoko/afkplus/releases
